src/components/card/LetterCard.jsx

```
@@ -8,9 +11,26 @@ const LetterCard = ({
   date,
   subject,
   onClick,
+  for_modify = false,
```

ESLint

[error] 'for_modify' is missing in props validation

(react/prop-types)